tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2082,6 +1818,43 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 		);
 	}
 
+	/**
+	 * @Then /^the JSON data of the response should (not )?contain (?:display|group) name "([^"]*)"(?: and match)?$/
+	 *
+	 * @param string $shouldOrNot (not|)
+	 * @param string $displayName
+	 * @param PyStringNode|null $schemaString
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theJsonDataResponseShouldContainDisplayNameAndMatch(
+		string $shouldOrNot,
+		string $displayName,
+		?PyStringNode $schemaString = null
```

Comment:
```suggestion
		PyStringNode $schemaString = null
```

Reply:
the single steps implementation supports 2 steps and the other steps we do not provide the schema so null.